global_optimizer_config.json

```diff
@@ -1,75 +1,82 @@
-    },
-    
-    "prob.perturbation": 0.59,
-    "prob.combine": 0.15,
-    "prob.split" : 0.25,
-    "perturbation":
-    {
-        "prob.x": 0.35,
-        "prob.y": 0.35,
-        "prob.width":0.1,
-        "prob.length":0.2,
-        "prob.rotation":0.2,
-        "modification.x.mu":0.0,
-        "modification.y.mu":0.0,
-        "modification.width.mu":0.0,
-        "modification.length.mu":0.0,
-        "modification.rotation.mu":0.0,
-        "modification.x.sigma":0.5,
-        "modification.y.sigma":0.5,
-        "modification.width.sigma":0.1,
-        "modification.length.sigma":0.2,
-        "modification.rotation.sigma":0.2,
-        "prob.opacity": 0.2,
-        "modification.opacity.mu":0.0,
-        "modification.opacity.sigma":0.001
-    },
-
-    "global_optimizer.window_size": 1,
-    "iteration_per_cell": 2000,
-    "auto_temp_scheduler.iteration_per_cell": 500,
-    "output.format": "png", //must be png or jpeg. If jpeg is chosen, then the output image quality can be set.
-    "output.quality": 80, // range [0, 100]
-    //residual.vmin must equal -residual.vmax
-    "residual.vmin": -0.3,
-    "residual.vmax": 0.3,
-    //objective parameter
-    "split.cost": 0.0,
-    "combine.cost": 0.5,
-    "overlap.cost": 0.2,
-    //cell.importance = importance of synthetic cell region/importance of synthetic diffraction region
-    "cell.importance": 1,
-    "prob.background_offset": 0.01, //This should be set to 0 if the algorithm runs in binary version
-    "background_offset.mu": 0.0,
-    "background_offset.sigma": 0.01,
-    "cell_brightness.mu": 0.0,
-    "cell_brightness.sigma": 0.01
-}
\ No newline at end of file
+{
+    // Global settings
+    "global.cellType": "bacilli",
+    "global.pixelsPerMicron": 1.0,
+    "global.framesPerSecond": 1.0,
+
+    // Bacilli settings
+    "bacilli.maxSpeed":  24.0,  // microns per second
+    "bacilli.maxSpin":    1.57, // radians per second
+    "bacilli.minGrowth": -2.0,  // microns
+    "bacilli.maxGrowth": 13.0,  // microns
+    "bacilli.minWidth":   4.0,  // microns
+    "bacilli.maxWidth":   9.0,  // microns
+    "bacilli.minLength": 14.0,  // microns
+    "bacilli.maxLength": 48.0,  // microns
+    "bacilli.distanceCostDivisor": 24,  // microns
+    
+    // simulation settings. NOTE: Image are converted to grayscale float32. 
+    // All settings should be floats.
+    "simulation":
+    {
+        "image.type": "To be Set", 
+        "background.color": "auto", // 0.39 for gary scale image
+        "cell.color" : "auto", // 0.15 for gray scale image
+        "light.diffraction.sigma": "auto",  //11
+        "light.diffraction.strength": "auto", //0.6
+        "light.diffraction.truncate": 3,
+        "cell.opacity": "auto"
+    },
+    
+    "prob.perturbation": 0.89,
+    "prob.combine": 0.05,
+    "prob.split" : 0.05,
+    "prob.opacity_diffraction_offset": 0.0,
+    "prob.background_offset": 0.01, //This should be set to 0 if the algorithm runs in binary version

```

Comment:
Make this automatic in the code; if (binary) set it to zero (print a notification as well to standard output)